Sources/StreamChat/Models/Attachments/ChatMessageAttachment.swift

```diff
@@ -4,8 +4,6 @@
 
 import Foundation
 
-public typealias ChatMessageAttachment = _ChatMessageAttachment<Any>
-
```

Comment:
I think we should keep this but rename it to `AnyChatMessageAttachment` to match the Swift naming style for type-erased types.